repo/repo-sql-impl/src/main/java/com/evolveum/midpoint/repo/sql/data/common/RCase.java

```diff
@@ -54,6 +54,7 @@
     private RPolyString nameCopy;
 
     private String state;
+    private String description;
```

Comment:
Do we really need to search by description? Maybe full text search would be a better solution.